newIDE/app/src/UI/Breadcrumbs.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import * as React from 'react';
+import MuiBreadcrumbs from '@material-ui/core/Breadcrumbs';
+
+import Text from './Text';
+import Link from './Link';
+
+type BreadcrumbStep =
+  | {| label: React.Node |}
+  | {| label: React.Node, onClick: () => void, href: string |};
+
+type Props = {|
+  steps: Array<BreadcrumbStep>,
+|};
+
+const Breadcrumbs = ({ steps }: Props) => {
```

Comment:
uh, where is this used?

Reply:
It's not used but it might be in the future. I can have it removed if you think that's too early

Reply:
I think it's fine, I was just wondering if it was a mistake :)